Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,135 @@
+      puts "cd #{formula.tap.path}"
+      cd formula.tap.path
+      puts "cp #{formula_path.relative_path_from(formula.tap.path)} #{boneyard_tap.formula_dir}"
+      puts "git rm #{formula_path.relative_path_from(formula.tap.path)}"
+      unless File.exist? tap_migrations_path
+        puts "Creating tap_migrations.json for #{formula.tap.name}"
+        puts "git add #{tap_migrations_path}"
+      end
+      puts "Loading tap_migrations.json"
+      puts "Adding #{formula.name} to tap_migrations.json"
+    else
+      safe_system HOMEBREW_BREW_FILE, "update"
+      safe_system HOMEBREW_BREW_FILE, "tap", boneyard_tap.name
+      cd formula.tap.path
+      cp formula_path.relative_path_from(formula.tap.path), boneyard_tap.formula_dir
+      safe_system "git", "rm", formula_path.relative_path_from(formula.tap.path)
+      unless File.exist? tap_migrations_path
+        tap_migrations_path.write <<-EOS.undent
+          {
+          }
+        EOS
+        safe_system "git", "add", tap_migrations_path
+      end
+      tap_migrations = Utils::JSON.load(File.read(tap_migrations_path))
+      tap_migrations[formula.name] = boneyard_tap.name
+      tap_migrations = tap_migrations.sort.inject({}) { |a, e| a.merge!(e[0] => e[1]) }
+      tap_migrations_path.atomic_write(JSON.pretty_generate(tap_migrations) + "\n")
+    end
+    unless Formula["hub"].any_version_installed?
+      if ARGV.dry_run?
+        puts "brew install hub"
+      else
+        safe_system HOMEBREW_BREW_FILE, "install", "hub"
+      end
+    end
+    branch = "#{formula.name}-boneyard"
+    puts "cd #{formula.tap.formula_dir}" if ARGV.dry_run?
+    cd formula.tap.formula_dir
+    if ARGV.dry_run?
+      puts "git checkout -b #{branch} origin/master"
+      puts "git commit --no-edit --verbose --message=\"#{formula.name}: migrate to boneyard\" -- #{formula_path} #{tap_migrations_path}"
+      puts "hub fork --no-remote"
+      puts "hub fork"
+      puts "hub fork (to read $HUB_REMOTE)"
+      puts "git push $HUB_REMOTE #{branch}:#{branch}"
+      puts "hub pull-request -m $'#{formula.name}: migrate to boneyard\\n\\nCreated with `brew boneyard-formula-pr`.'"
+    else
+      safe_system "git", "checkout", "-b", branch, "origin/master"
+      safe_system "git", "commit", "--no-edit", "--verbose",
+        "--message=#{formula.name}: migrate to boneyard",
+        "--", formula_path, tap_migrations_path
+      safe_system "hub", "fork", "--no-remote"
+      quiet_system "hub", "fork"
+      remote = Utils.popen_read("hub fork 2>&1")[/fatal: remote (.+) already exists./, 1]
+      odie "cannot get remote from 'hub'!" unless remote
+      safe_system "git", "push", remote, "#{branch}:#{branch}"
+      pr_message = <<-EOS.undent
+        #{formula.name}: migrate to boneyard
+
+        Created with `brew boneyard-formula-pr`
+      EOS
+      pr_url = Utils.popen_read("hub", "pull-request", "-m", pr_message).chomp
+    end
+    puts "cd #{boneyard_tap.formula_dir}" if ARGV.dry_run?
```

Comment:
This doesn't print anything after the `cd` in dry-run mode and when `homebrew/boneyard` hasn't been tapped yet.